charts/argo/templates/server-deployment.yaml

```diff
@@ -52,7 +52,11 @@ spec:
             httpGet:
               path: /
               port: 2746
+              {{- if has "--secure" .Values.server.extraArgs }}
```

Comment:
I wonder if instead we want a value on the chart for turning on secure mode, i.e. `server.secure=true`, this can then add the `--secure` flag and change the scheme of the probe, doing magic things based on `extraArgs` never sits right with me.

Reply:
I would be fine with that